spec/models/group_spec.rb

```diff
@@ -725,6 +725,10 @@ def can_view?(user, group)
         .and change { user.title }.from('AAAA').to('BBBB')
     end
 
+    it "can send a notification to the user" do
+      expect { group.add(user, notify: true) }.to change { Notification.count }.by(1)
```

Comment:
We don't want to just check for the count. The attributes of the notification are important as well.